packit_service/worker/reporting.py

```diff
@@ -5,10 +5,10 @@
 from enum import Enum
 from typing import Optional, Union, Dict
 
-import github
 import gitlab
```

Comment:
We should probably look into the reason for `gitlab` being here later on…

Reply:
similarly as for Github, we are catching `gitlab.exceptions.GitlabCreateError` for creating statuses and fallback to comments but if I checked the ogr PR correctly, this exceptions should not be wrapped, am I right?

Reply:
iirc we wrap both now… there is an easy way to find out tbh :D

Reply:
ok, will add it

Reply:
ya, it is…

_mf linking the Sentry issues_